tests/Branch.py

```diff
@@ -67,6 +67,7 @@ def testEditProtection(self):
         self.assertTrue(branch_protection.required_status_checks.strict)
         self.assertEqual(branch_protection.required_status_checks.contexts, [])
         self.assertTrue(branch_protection.enforce_admins)
+        self.assertFalse(bool(branch_protection.required_linear_history))
```

Comment:
Shouldn't this already be a `bool`? Why the need to cast it?

Reply:
Fixed in 08546365d5160dd63360af8a4d21d8e6d269bde8